.github/workflows/build.yml

```diff
@@ -19,40 +15,35 @@ jobs:
     strategy:
       matrix:
         image:
-        - debug
-        - debian10
-        - fedora37
-        - fedora37-test
-        - fedora38
-        - fedora39
         - alma8
         - alma9
-        - ubuntu2310
-        - ubuntu2304
-        - ubuntu22
+        - fedora38
+        - fedora39
         - ubuntu20
-        - ubuntu18
+        - ubuntu22
+        - ubuntu2304
```

Comment:
Isn't 23.04 already past its EOL? If yes, maybe we can get rid of it?

Reply:
Yes indeed, dropped as well :+1:

Reply:
Thanks. Let's merge once the builds are green.